src/fadec/a380_fadec/src/FadecGauge.cpp

```diff
@@ -0,0 +1,32 @@
+#include "FadecGauge.h"
+
+FadecGauge FADEC_GAUGE;
+
+__attribute__((export_name("FadecGauge_gauge_callback"))) extern "C" bool FadecGauge_gauge_callback(FsContext ctx,
```

Comment:
Instead of `__attribute__((export_name("FadecGauge_gauge_callback"))) ` you could also just use `MSFS_CALLBACK`. Nothing huge, just looks a bit better :D

Reply:
It's been a while, but I recall Andreas giving me the tip to change it long time ago. I recall it made sense, but to be honest cannot tell you why :)